navigator/backend/src/main/scala/com/digitalasset/navigator/json/ApiCodecCompressed.scala

```diff
@@ -53,6 +58,14 @@ object ApiCodecCompressed {
   def apiRecordToJsValue(value: Model.ApiRecord): JsValue =
     JsObject(value.fields.map(f => f.label -> apiValueToJsValue(f.value)).toMap)
 
+  private[this] val fieldKey = "key"
+  private[this] val fieldValue = "value"
+
+  def apiMapToJsValue(value: Model.ApiMap): JsValue =
```

Comment:
as discussed, please convert it to an object directly, e.g.

```
{
  k1: v1,
  k2: v1,
  ...
}
```

Reply:
fixed